configure.ac

```diff
@@ -99,7 +100,11 @@ dnl ---------------------------------------------------------------------------
 dnl - Check library dependencies
 dnl ---------------------------------------------------------------------------
 PKG_CHECK_MODULES(GLIB, glib-2.0 >= 2.44.0 gobject-2.0 gthread-2.0 gio-2.0)
-PKG_CHECK_MODULES(USB, libusb-1.0 >= 1.0.19)
+case "$host_os" in
+	freebsd*) libusb_version="1.0.9" ;;
```

Comment:
Err, I'm a bit confused. If FreeBSD provides a different library (with a different version->ABI map) surely it should be called something different to `libusb-1.0`? That seems wrong at best...

Reply:
It is a different library which is usually installed as /usr/lib/libusb.so.3. It provides 3 .pc files: libusb-0.1.pc, libusb-1.0.pc, libusb-2.0.pc. The version from http://libusb.info/ doesn't support FreeBSD.

Reply:
That's very misleading. If I have `libusb-1.0 >= 1.0.19` in my project that's requesting a set of ABI and API from a specific project. I understand if FreeBSD chooses to just mirror the API with the same pkg-config name if it's 100% compatible between each version, but then it has to be 100% compatible for each version, not just "similar".

Reply:
Yes, I can open a feature request for missing functions on FreeBSD bugzilla, but it can take months or years for users to get the update even if it is accepted and implemented immediately, depending on whether the patch is picked for backporting to stable branches.

Reply:
Got it. Lets see what can be done!